dangerzone/conversion/pixels_to_pdf.py

```diff
@@ -11,12 +11,16 @@
 import os
 import shutil
 import sys
+from typing import Optional
 
 from .common import DangerzoneConverter, running_on_qubes
 
 
 class PixelsToPDF(DangerzoneConverter):
-    async def convert(self) -> None:
+    async def convert(self, ocr_lang: Optional[str] = None) -> None:
+        ocr_enable = ocr_lang is not None or os.environ.get("OCR") == "1"
```

Comment:
I would only add a comment here because for future people taking a look at this code it may be weird why to have a parameter but also obtain it from an env var
```suggestion
        # XXX Qubes converter uses the parameter, while the containers version passes via env vars
        ocr_enable = ocr_lang is not None or os.environ.get("OCR") == "1"
```

Reply:
Actually, I think we could have a better approach, if you agree. We could move this env var logic onto `main()` [down in the file](https://github.com/freedomofpress/dangerzone/pull/544/files#diff-bdfbfeb3ce16c19ed296150fa806b7ac3165fa32aa249a3225ec54d70594888eR163) (that code is only executed in containers).

Then this would always receive some `ocr_lang` (or None). Furthermore, we can drop the `ocr_enable` variable and just keep the `ocr_lang`. If not `None` then do OCR with that lang, if not we drop it.

Reply:
Damn, you're totally right! That's indeed a great suggestion. I've fixed it in a6f65de.